bcda/api.go

```diff
@@ -123,12 +123,24 @@ func jobStatus(w http.ResponseWriter, r *http.Request) {
 
 		jsonData, err := json.Marshal(rb)
 		if err != nil {
+			errorStatus := createOperationOutcome("marshalling error")
+			_, err = w.Write([]byte(errorStatus))
+			if err != nil {
+				http.Error(w, http.StatusText(500), 500)
+				return
+			}
 			http.Error(w, http.StatusText(500), 500)
 			return
 		}
 
 		_, err = w.Write([]byte(jsonData))
 		if err != nil {
+			errorStatus := createOperationOutcome("writing error")
+			_, err = w.Write([]byte(errorStatus))
+			if err != nil {
```

Comment:
Are lines 140-142 redundant?  Looks like we are doing the following regardless:
`http.Error(w,` http.StatusText(500), 500)
return`

Reply:
Same deal as the above comment